sql/core/benchmarks/DataSourceReadBenchmark-jdk21-results.txt

```diff
@@ -2,430 +2,430 @@
 ---------------------------------------------------------------------------------------------------------------------------
-ParquetReader Vectorized: DataPageV1                   130            140          11        121.1           8.3       1.0X
-ParquetReader Vectorized: DataPageV2                   213            230          10         74.0          13.5       0.6X
-ParquetReader Vectorized -> Row: DataPageV1            128            132           6        122.9           8.1       1.0X
-ParquetReader Vectorized -> Row: DataPageV2            222            226           5         70.7          14.1       0.6X
+ParquetReader Vectorized: DataPageV1                   150            157           6        104.7           9.6       1.0X
+ParquetReader Vectorized: DataPageV2                   212            226           9         74.3          13.5       0.7X
+ParquetReader Vectorized -> Row: DataPageV1            164            170           6         95.8          10.4       0.9X
+ParquetReader Vectorized -> Row: DataPageV2            242            246           4         64.9          15.4       0.6X
 
-OpenJDK 64-Bit Server VM 21.0.2+13-LTS on Linux 6.5.0-1016-azure
+OpenJDK 64-Bit Server VM 21.0.3+9-LTS on Linux 6.5.0-1018-azure
 AMD EPYC 7763 64-Core Processor
 SQL Single BIGINT Column Scan:            Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-SQL CSV                                           14617          14690         103          1.1         929.3       1.0X
-SQL Json                                          10772          10780          11          1.5         684.9       1.4X
-SQL Parquet Vectorized: DataPageV1                  118            132          13        133.4           7.5     124.0X
-SQL Parquet Vectorized: DataPageV2                  268            300          20         58.7          17.0      54.5X
-SQL Parquet MR: DataPageV1                         2289           2314          36          6.9         145.5       6.4X
-SQL Parquet MR: DataPageV2                         1993           1995           3          7.9         126.7       7.3X
-SQL ORC Vectorized                                  215            224          12         73.1          13.7      68.0X
-SQL ORC MR                                         1840           1851          17          8.6         117.0       7.9X
-
-OpenJDK 64-Bit Server VM 21.0.2+13-LTS on Linux 6.5.0-1016-azure
+SQL CSV                                           11095          11134          54          1.4         705.4       1.0X
+SQL Json                                           9688           9701          18          1.6         616.0       1.1X
+SQL Parquet Vectorized: DataPageV1                  293            297           4         53.7          18.6      37.9X
```

Comment:
`DataPageV1` seems to show a regression in `BIGINT` column. It's slower than `DataPageV2` of Parquet.